common/lvgl/src/lv_font/fcc_logo_black_2020.c

```diff
@@ -0,0 +1,99 @@
+#ifdef __has_include
```

Comment:
1. Add copyright information in all new .c and .h files
2. Double check if these checks are really required

Reply:
Added the copyright statements

Reply:
This file was also auto-generated by lvgl font converter online
Don't see any harm in keeping the checks, while it builds without it too, the LV_LVGL_H_INCLUDE_SIMPLE macro is used in other files as well.